examples/diagnostic_protocol/README.md

```diff
@@ -0,0 +1,4 @@
+cd ISO11783-CAN-Stack
```

Comment:
There is no README currently  on how to build examples, if they are build separately, then this is the process for each of em. The advantage of building examples separately like this is it demonstrates how someone will actually consume the installed lib vs a  local build tree. It can be used to test the install if set up correctly in CI.

Reply:
Added a README in 8c1e61e76c2fb0bb0416e32edfc16f0d1e73659b